### PR TITLE
[BACKLOG-6764, BACKLOG-6885] - RV: Reviewing hideProgressIndicator an…

### PR DESCRIPTION
**Pull request by jvelasques**
…d showProgressIndicator usages

### CODE REVIEW

**Comment by jvelasques** [comment]
@pentaho/orlandocalisbon please review

**Comment by lucboudreau** [comment]
http://10.100.11.222:8080/job/wingman-osgi/61/

**Comment by lucboudreau** [comment]
Build Commands:
==============
Build:
------
1. ant -Dtestreports.dir=bin/reports/unit-test -Djunit.jvmargs=-XX:PermSize=256m -f build.xml clean-all resolve jacoco && ant -Dpentaho.coding.standards.version=1.0.4 -Dtestreports.dir=bin/reports/integration-test -Djunit.jvmargs=-XX:PermSize=256m -f build.xml jacoco-integration test-js checkstyle

Cleanup:
-------
rm -r ~/.ivy2/local || echo no publish local to remove



Changed files:
==============
  package-res/reportviewer/reportviewer-prompt.js
  test-js/reportviewer-prompt-spec.js

File: package-res/reportviewer/reportviewer-prompt.js [hunk 1]
```diff
@@ -368,12 +368,12 @@ define(['common-ui/util/util', 'pentaho/common/Messages', "dijit/registry", 'com
         if (blocker) {
           messageBox.setButtons([]);
         } else {
-          var closeFunc = function() {
-            if (this.panel) {
-              this.panel.dashboard.hideProgressIndicator();
+          var closeFunc = (function() {
+            if(this.panel) {
```
Comment (annapopova1) [inline package-res/reportviewer/reportviewer-prompt.js]: Need try to remove this if statement - we should avoid using prompt panel object directly.

File: package-res/reportviewer/reportviewer-prompt.js [hunk 2]
```diff
@@ -397,8 +397,8 @@ define(['common-ui/util/util', 'pentaho/common/Messages', "dijit/registry", 'com
             messageBox.setButtons([button1Text]);
           }
         }
-        if (this.panel) {
-          this.panel.dashboard.showProgressIndicator();
+        if(this.panel) {
```
Comment (annapopova1) [inline package-res/reportviewer/reportviewer-prompt.js]: Also the same comment for this if statement

**Pull request merged**